### PR TITLE
Add node methods for node selection

### PR DESCRIPTION
**Pull request by lascavana**
This pull request adds a pure wrapper of the following SCIP functions:
 “SCIPgetBestChild”,  "SCIPgetBestSibling” ,  “SCIPgetBestLeaf”,  “SCIPgetBestNode”,  "SCIPgetBestboundNode" from "scip/scip_tree.h".

### CODE REVIEW

File: src/pyscipopt/scip.pyx [hunk 1]
```diff
@@ -1657,6 +1657,46 @@ cdef class Model:
         PY_SCIP_CALL(SCIPupdateNodeLowerbound(self._scip, node.scip_node, lb))
 
     # node methods
+    def getBestChild(self):
+        """gets the best child of the focus node w.r.t. the node selection strategy."""
+        cdef SCIP_NODE* _node
+
+        _node = SCIPgetBestChild(self._scip)
+        node = Node.create(_node)
+        return node
```
Comment (fserra) [inline src/pyscipopt/scip.pyx:1666]: Thanks! could you rewrite it in a single line, please? Like
`return Node.create(SCIPgetBestChild(self._scip))`

**Comment by fserra** [comment]
cool! thanks!

**Comment by fserra** [comment]
@mattmilten do you have a clue what this madness with windows is?

**Comment by mattmilten** [comment]
Oh boy, I have no idea yet what this is. It also does not seem to have anything to do with this specific pull request.

**Comment by mattmilten** [comment]
Travis jobs are also failing. This is not a Windows problem

**Pull request merged**